test/rest_api/test_system.py

```diff
@@ -11,3 +14,17 @@ def test_system_highest_sstable_format(rest_api):
     resp = rest_api.send('GET', "system/highest_supported_sstable_version")
     resp.raise_for_status()
     assert resp.json() == "me"
+
+@pytest.mark.parametrize("params", [
+    ("storage_service/flush_throughput", "mibs"),
+    ("commitlog/throughput", "mibs"),
+    ("storage_service/compaction_throughput", "value"),
+    ("storage_service/stream_throughput", "value")
+])
+def test_set_throughput(rest_api, params):
+    resp = rest_api.send("POST", params[0], params={ params[1]: 100 })
+    resp.raise_for_status()
+    resp = rest_api.send("POST", params[0], params={ params[1]: 0 })
+    resp.raise_for_status()
```

Comment:
Should also read back the value with `GET` variant to make sure it was updated.

Reply:
There's no get for commitlog and flush
But in general I agree, compaction and streaming have getter and can be checked, will add